xml/geo_booth_i.xml

```diff
@@ -104,19 +173,15 @@ site =  192.168.201.100 <co xml:id="co.ha.geo.booth.config.site"/>
 site =  192.168.202.100 <xref linkend="co.ha.geo.booth.config.site" xrefstyle="select:label nopage"/>
 authfile = /etc/booth/authkey <co xml:id="co.ha.geo.booth.config.auth"/>
 ticket = "ticket-nfs" <co xml:id="co.ha.geo.booth.config.ticket"/>
+     retries = 5 <co xml:id="co.ha.geo.booth.config.retries"/>
+     mode = MANUAL <co xml:id="co.ha.geo.booth.mode"/>
```

Comment:
@ChrisKowalczyk IIUC "retries" is related to renewal of tickets, and manual tickets don't need to be renewed, probably "retries" doesn't need to be configured here?

Reply:
Yes, we discussed this already with Chris and a little bit later in the text, it is mentioned  that manual tickets don't have this option. Removed from here with https://github.com/SUSE/doc-sleha/commit/d90d90c3.